ultrasonic/src/main/kotlin/org/moire/ultrasonic/playback/PlaybackService.kt

```diff
@@ -48,18 +47,6 @@ class PlaybackService : MediaLibraryService(), KoinComponent {
      * For some reason the LocalConfiguration of MediaItem are stripped somewhere in ExoPlayer,
      * and thereby customarily it is required to rebuild it..
      */
-    private class CustomMediaItemFiller : MediaSession.MediaItemFiller {
-        override fun fillInLocalConfiguration(
-            session: MediaSession,
-            controller: MediaSession.ControllerInfo,
-            mediaItem: MediaItem
-        ): MediaItem {
-            // Again, set the Uri, so that it will get a LocalConfiguration
-            return mediaItem.buildUpon()
-                .setUri(mediaItem.mediaMetadata.mediaUri)
-                .build()
-        }
-    }
 
```

Comment:
Nice, this is not needed anymore?
It was a weird bit of architecture...

But then the comment can be removed as well

Reply:
From what I understood it is now directly done in onAddMediaItems(AutoMediaBrowserCallback.kt:184) but I'm not really sure it's necessary since it's also setup in MediaPlayerController.kt:662